Umweltzone/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,375 @@
+        Adicionalmente, hay enlaces a páginas web con más información sobre LEZ, DPZ y otros temas relacionados.
+    </string>
+
+    <!-- AboutActivity / Disclaimer -->
+    <string name="appinfo_disclaimer_title">Datos, mediciones y actualizaciones</string>
+    <string name="appinfo_disclaimer_content">
+        Siempre que los datos y las coordenadas de las zonas estuvieran disponibles en un formato digital,
+        fueron obtenidas directamente de las ciudades or los goviernos.
+        Además, OpenStreetMap fue una importante fuente de datos, ya que hay un número de
+        zonas de baja emisión que pueden ser encontradas allí. Adicionalmente, algunas zonas se
+        digitalizaron manualmente basado en las publicaciones oficiales de la Agencia Federal Del Ambiente.
+        Debido al hecho de muchas de la autoridades publican gráficos escaneados a un baja resolución, no
+        se puede garantizar que los límites coincidan exactamente con la ruta en la realidad.
+        Los datos de las zonas y las coordenadas pueden variar según la fuente,
+        calidad y presición en el proceso de digitalización
+        Además, no hay un mecanismo para automáticamente informar sobre los cambios hecho en cada ciudad.
+        Los datos de la zona son actualizados manualmente y son altamente dependientes de la retroalimentación
+        de los usuarios amigables como tú.
+        Por esa razón, no garantizamos una visualización perfecta y no podemos hacernos cargo de costos o sanciones.
+        Recomendamos encarnecidamente siempre confiar en las señales de tráfico de las ciudades en paticular.
+    </string>
+    <string name="appinfo_references_name_umweltbundesamt">Agencia Federal del Ambiente</string>
+
+
+    <!-- AboutActivity / Graphics -->
+    <string name="appinfo_references_graphics_title">Gráficos</string>
+    <string name="appinfo_references_graphics_content">
+        Gráficos como el logo y las etiquetas fueron tomados de Wikimedia Commons.
+    </string>
+
+
+    <!-- AboutActivity / Texts -->
+    <string name="appinfo_references_texts_title">Textos</string>
+    <string name="appinfo_references_texts_content">
+        Se proporciona una referencia de la fuente cada vez que se toma texto de otro lugar.
+    </string>
+
+    <!-- AboutActivity / Libraries -->
+    <string name="appinfo_references_libraries_title">Librerías</string>
+    <string name="appinfo_references_libraries_content">
+        Las siguientes librerías de software son usadas en esta aplicación.
+    </string>
+
+    <!-- AboutActivity / Sourcecode -->
+    <string name="appinfo_sourcecode_title">Código fuente</string>
+    <string name="appinfo_sourcecode_content">
+        El código fuente de la aplicación está disponible en línea.
+    </string>
+    <string name="appinfo_sourcecode_url_title">Código fuente de la aplicación</string>
+
+
+    <!-- AboutActivity / License -->
```

Comment:
Please also add a string for `appinfo_license_url_title_creative_commons`.

Reply:
to close of no translatable items xD

Reply:
The word "licence" is different in some languages.